modules/setting/cors.go

```diff
@@ -12,9 +12,7 @@ import (
 // CORSConfig defines CORS settings
 var CORSConfig = struct {
 	Enabled          bool
-	Scheme           string
-	AllowDomain      []string
-	AllowSubdomain   bool
+	AllowDomain      []string // this option is from legacy code, it should be called "AllowedOrigins" actually
```

Comment:
```suggestion
	AllowedOrigins      []string `ini:"ALLOW_DOMAIN"`
```
If I'm not mistaken.

Reply:
I don't like introducing this change too early.

That option should be refactor later totally, but not just rename it here.

Even if you rename it here, it is still inconsistent from config (why AllowedOrigins vs ALLOW_DOMAIN)

Reply:
```suggestion
	AllowDomain      []string // FIXME: this option is from legacy code, it actually works as "AllowedOrigins". When refactoring in the future, the config option should also be renamed together.
```